frontend/www/js/omegaup/components/contest/Edit.vue

```diff
@@ -303,6 +299,20 @@ export default class Edit extends Vue {
     }
   }
 
+  get buttonDescription(): string {
```

Comment:
```suggestion
  get archiveButtonDescription(): string {
```

por consistencia con el de abajo.